### DISCUSSION TITLE
News & Ecommerce Themes

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#1521] [Q&A] [answered]
Hi,

I'm awaiting for Oqtane. 

1. Current version is production ready?
2. Any themes available **"News & Ecommerce Themes"**?

**Comment by sbwalker** [comment, marked as answer]
1. Yes the current version is production ready
2. There are limited third party themes currently available - have you seen the Bootswatch theme collection: https://github.com/oqtane/oqtane.theme.bootswatch
3. In regards to (2) an Oqtane "theme" provides the look and feel - not the functionality of an application. If you are looking for news article or ecommerce functionality then you are probably looking for a "module". Oqtane is relatively new so there are limited modules available - but you could certainly build your own.

**Reply by Sharafudeen** [reply]
I would like to build Modules like news, article, and ecommerce item list (such as books, etc.,) with cart and payment modules such as COD, Paypal, etc.,
Also in the article or news content management, I would like to attach audio & video files and option for player to play audio & video files.
Where can I find the module? Any links please..

Thanks!

**Reply by sbwalker** [reply]
@Sharafudeen you said that you would like to "build" modules - which I assume means that you would like to develop them yourself ( ie. write the code ). You can absolutely develop such modules on Oqtane - you just need to become familiar with the development techniques. I would suggest you start with the development video series on YouTube or use the Module Creator to create the foundation of your first module.

**Reply by ADefWebserver** [reply]
@Sharafudeen You can also find Oqtane Module Development documentation at: https://blazorhelpwebsite.com/

![image](https://user-images.githubusercontent.com/1857799/124035141-00c0b480-d9b1-11eb-80e7-8ab1128e0bdf.png)

**Reply by Sharafudeen** [reply]
@sbwalker @ADefWebserver Thanks for the link. I'm looking for ecommerce & blogger free or premier templates or themes for Oqtane. If anything available please share it. Thanks!
